tests/unit/test_k8s_resources.py

```diff
@@ -75,6 +75,7 @@ def test_create_template_values(valid_spec):
         "path",
         "pvc",
         "routing",
+        "schedulerName",
```

Comment:
```suggestion
        "scheduler_name",
```
according to my other suggestions